source/variables/z-index.css

```diff
@@ -0,0 +1,12 @@
+/* 
+The z-indexes for the project. 
+z-index: var(--navIndex);
+*/
+:root {
+	--index--hidden: -100;
+	--index--base: 1;
+	--index--content: 10;
+	--index--nav: 50;
+	--index--tooltip: 1000;
```

Comment:
Don't stair directly into the 999999!
Agree, I'm not even sure why we are using such large distance numbers. Might of been older browser related. I'll update.